erpnext/buying/doctype/request_for_quotation/request_for_quotation.py

```diff
@@ -226,3 +227,16 @@ def create_rfq_items(sq_doc, supplier, data):
 		"request_for_quotation_item": data.name,
 		"request_for_quotation": data.parent
 	})
+
+@frappe.whitelist()
+def download_pdf(doctype, name, supplier_idx):
+	doc = get_rfq_doc(doctype, name, supplier_idx)
+	if doc:
+		frappe.get_attr("frappe.www.print.download_pdf")(doctype, name, doc=doc)
```

Comment:
Yeah! Do this - move all the logic to another file, maybe `frappe/utils/print_format.py` and in print.py just keep the `get_context` function.